src/azure-cli/setup.py

```diff
@@ -119,7 +119,7 @@
     'azure-mgmt-search~=9.0',
     'azure-mgmt-security==3.0.0',
     'azure-mgmt-servicebus~=8.2.0',
-    'azure-mgmt-servicefabricmanagedclusters~=1.0.0',
+    'azure-mgmt-servicefabricmanagedclusters~=2.0.0b4',
```

Comment:
For the beta version, we usually recommend fixing the version to avoid unexpected breaking changes caused by sudden upgrades of beta version


```suggestion
    'azure-mgmt-servicefabricmanagedclusters==2.0.0b4',
```